tests/filesystem-sensitivity/src/test/kotlin/MainTest.kt

```diff
@@ -0,0 +1,45 @@
+import kotlin.test.Test
+
+class MainTest {
+  @Test
+  fun test() {
+
+    val os = System.getProperty("os.name").lowercase()
+    if (os.contains("mac")) {
```

Comment:
Won't work on Macs with case-sensitive FS but we could use/duplicate `isFileSystemCaseSensitive` if that ever happens to be a problem :)

Reply:
I was hoping that both Github MacOS runners and most dev laptops would use case insensitive filesystems. The good thing is that if it's the case, is also tests `isFileSystemCaseSensitive`. 🤞 it'll be the case. If not I'll find another test I guess.